GettingStarted.md

```diff
@@ -0,0 +1,181 @@
+  * [Mac: Apple Java 1.6.0_65, a.k.a. Java for OS X 2015-001](https://support.apple.com/kb/DL1572?locale=en_US)
+1. Set the JAVA_HOME environment variable to point to the install, e.g
+ * (Windows) `SET JAVA_HOME=C:\Program Files\Java\jdk1.6.0_45`
+ * (Linux) `JAVA_HOME=~/dev/java/jdk1.6.0_45`
+ * (Mac) `JAVA_HOME=/Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home`
+1. Add JAVA_HOME bin directory to the path
+ * (Windows) `SET PATH=%JAVA_HOME%\bin;%PATH%`
+ * (Linux) `PATH=$JAVA_HOME/bin:$PATH`
+ * (Mac) `PATH=$JAVA_HOME/bin:$PATH`
+1. Download and install Java SE 8 JDK from [Java SE Downloads](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+
+### Install Ant
+
+1. If you do not already have it, download and install Apache Ant(TM) version 1.9.6 from [Ant Binary Distributions](http://ant.apache.org/bindownload.cgi).
+1. Add the full path of the Ant `bin` directory to the `PATH` system environment variable. You can find more Ant installation details [here](http://ant.apache.org/manual/install.html#installing).
+ * (Windows) `SET PATH=C:\dev\apache-ant-1.9.6\bin;%PATH%`
+ * (Linux) `PATH=~/dev/apache-ant-1.9.6/bin:$PATH`
+ * (Mac) `PATH=~/dev/apache-ant-1.9.6/bin:$PATH`
+
+### Install Eclipse
+
+Two versions of Eclipse are needed: we use Eclipse 3.5.2 (Galileo) as the Target Environment, which is the minimum supported Eclipse version (used for building the release and testing backward-compatibility) and Eclipse 4.5.2 (Mars.2) as a dedicated team-explorer-everywhere Development Environment.  Using a dedicated Eclipse installation and workspace for development makes it less disruptive to configure Eclipse-wide preferences.
+
+1. Download and extract [Eclipse Classic 3.5.2](http://www.eclipse.org/downloads/packages/eclipse-classic-352/galileosr2) to the `dev/eclipseTargets/352` sub-folder under your `HOME` folder.  The absolute path to this folder will be needed later when building with Ant.
+  * (Windows) Use [7-Zip](http://www.7-zip.org/) to [make sure all files are extracted](https://bugs.eclipse.org/bugs/show_bug.cgi?id=166597).
+1. Launch Eclipse 3.5.2
+  1. `Help` -> `Install New Software...`
+  1. In the *Work with* textbox, type `http://archive.eclipse.org/egit/updates-2.1` and click `Add...`
+  1. The *Add Site* dialog appears.  Type `EGit 2.1.0` in the _Name_ textbox and click `OK`.
+  1. Expand both features and select the following plug-ins:
+    * *Eclipse EGit*
+    * *Eclipse EGit - Source*
+    * *Eclipse JGit*
+    * *Eclipse JGit - Source*
+  1. Click `Next >` and finish the wizard to install the plugins
+  1. TODO: install more modern SWT than 4.1 for browser-based authentication
+1. Download and extract [Eclipse for RCP and RAP Developers 4.5.2](http://www.eclipse.org/downloads/packages/eclipse-rcp-and-rap-developers/mars2)
+  * (Windows) Use [7-Zip](http://www.7-zip.org/) to [make sure all files are extracted](https://bugs.eclipse.org/bugs/show_bug.cgi?id=166597).
```

Comment:
Hmmm... If we don't point out the potential defect (which may, in fact, occur only in older versions of Windows) and give freedom to the end-user to extract files anywhere (i.e. the "hardcoded" comment earlier), we may be setting them up for failure.  I'd rather they not risk having a broken Eclipse installation if they somehow can't create folders at the root of the system drive and assume they can just extract somewhere else.